scipy/linalg/_basic.py

```diff
@@ -510,12 +514,18 @@ def solveh_banded(ab, b, overwrite_ab=False, overwrite_b=False, lower=False,
     Returns
     -------
     x : (M,) or (M, K) ndarray
-        The solution to the system a x = b. Shape of return matches shape
+        The solution to the system ``a x = b``. Shape of return matches shape
         of `b`.
 
+    Notes
+    -----
+    In the case of a nonpositive matrix ``a``, the solver `solve_banded`
+    may be used.
+
```

Comment:
```suggestion
    In the case of a non-positive definite matrix ``a``, the solver
    `solve_banded` may be used.
```

Reply:
Apologies for the delay. Note that it can be used for nonpositive arrays but the result is not guaranteed. But that's rather a minor point I don't have any problems with as is. Thanks for sweeping these.

Reply:
I tried it for the example problem in the documentation but replacing the `[0, 0]` entry (`4`) with `-1`, which makes one of the eigenvalues negative. I got
```
  File ~\Desktop\scipy\scipy\linalg\_basic.py:584 in solveh_banded
    raise LinAlgError("%dth leading minor not positive definite" % info)

LinAlgError: 1th leading minor not positive definite
```
so I think I'll leave it as-is. After all, it doesn't say it _can't_ be used with non-positive definite input - more that it shouldn't.